src/main/java/com/github/thedeathlycow/moregeodes/features/ModPlacedFeatures.java

```diff
@@ -20,6 +21,7 @@ public class ModPlacedFeatures {
 
     public static final RegistryEntry<PlacedFeature> LAPIS_GEODE;
     public static final RegistryEntry<PlacedFeature> GYPSUM_PATCH;
+    public static final RegistryEntry<PlacedFeature> CERTUS_GEODE;
```

Comment:
This should also be marked as `@Nullable` explicitly

Reply:
True